satpy/readers/seviri_base.py

```diff
@@ -449,7 +449,12 @@ def get(self):
             ('TerminationSpace', 'S1'),
             ('EncodingVersion', np.uint16),
             ('Channel', np.uint8),
-            ('Filler', 'S20'),
+            ('ImageLocation', 'S3'),
```

Comment:
Is there a way to test these changes directly? At the moment they are only tested indirectly (there's a simplified version of that data type in the test).